libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.html

```diff
@@ -2,11 +2,15 @@
   <label
     *ngIf="labelText"
     class="sky-control-label sky-margin-inline-xs"
+    [ngClass]="{
+      'sky-screen-reader-only': labelHidden
+    }"
     [for]="inputId"
     >{{ labelText }}</label
   >
   <sky-help-inline
-    *ngIf="helpPopoverContent && labelText"
+    *ngIf="(helpPopoverContent || helpKey) && labelText && !labelHidden"
```

Comment:
Missing unit test for scenario where `labelText` is provided but `labelHidden` is true for both `helpPopoverContent` and `helpKey`

Reply:
Added